networking/configuring-network-operator.adoc

```diff
@@ -0,0 +1,60 @@
+// Assembly filename: configuring-network-operator.adoc
+// Explains network operator, shows default CRDs and modifications to setup.
+[id='configuring-network-operator’]
+= Configuring the Network at Installation Time
+include::modules/common-attributes.adoc[]
+:context: configuring-network-operator
+
+toc::[]
+
+The Network Operator installs and upgrades the networking components
+on an {product-title} cluster.
+The Network Operator runs early in the cluster install process.
+If you want to customize the network configuration,
+you must modify the settings before installing the cluster.
+
+The Network Operator is configured from two objects: the
+'Cluster configuration' and the 'Operator configuration'.
+
+* The 'Cluster configuration' object is defined by the
+'Network.config.openshift.io' Custom Resource Definition (CRD).
+* The 'Operator configuration' object is defined by the
+'NetworkConfig.networkoperator.openshift.io' CRD.
+{product-title} generates the 'Operator configuration' from the 'Cluster
+configuration'.
+
+Once a cluster is running, it is not safe to change most of the network
+settings.
+The Network Operator prevents unsafe changes.
+
+//include::modules/configuration-resource-overview.adoc[leveloffset=+1]
+//
+//include::modules/networking-cluster-configuration.adoc[leveloffset=+1]
+
+//Basic network configuration defaults in network/cluster config
+include::modules/nw-network-config-crd.adoc[leveloffset=+1]
+
+//Advanced network configuration defaults in operator config
+include::modules/nw-networkoperator-crd.adoc[leveloffset=+1]
+
+//Modifying basic network configuration
+include::modules/nw-modifying-network-config-startup.adoc[leveloffset=+1]
+//
+//Modifying advanced network configuration at startup
+include::modules/nw-modifying-nwoperator-config-startup.adoc[leveloffset=+1]
+
+//Modifying kubeProxy configuration at startup
+include::modules/nw-nwop-config-kubeproxy.adoc[leveloffset=+2]
+
+//Modifying OpenShiftSDN configuration at startup
+include::modules/nw-nwop-config-OpenShiftSDN.adoc[leveloffset=+2]
+
+//Viewing network configuration in a running cluster
+include::modules/nw-nwop-viewing-config.adoc[leveloffset=+1]
+
+//Modifying network configuration in a running cluster
+include::modules/nw-nwop-modifying-config-running.adoc[leveloffset=+1]
+//
+//Full operator config. Network operator uses 2 CRDs, so separated them into 2 docs.
+//include::modules/network-annotated-crd.adoc[leveloffset=+1]
```

Comment:
For each of the commented-out include statements, try replacing "::" with "  ". That seems to make Travis happy.